Dockerfile-alpine.template

```diff
@@ -98,7 +101,9 @@ RUN set -eux; \
 # "/usr/src/postgresql/src/backend/access/common/tupconvert.c:105: undefined reference to `libintl_gettext'"
 #		--enable-nls \
 		--enable-integer-datetimes \
+{{ if .major <= 16 then ( -}}
 		--enable-thread-safety \
```

Comment:
From the doc, this feature switch to "on by default" and now has a disable flag so this is not needed if version 17

https://www.postgresql.org/docs/current/install-make.html#CONFIGURE-OPTION-DISABLE-THREAD-SAFETY